File: SmartTasks/Models/SmartTask.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace SmartTasks.Models
{
    public class SmartTask
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public DateTime CreateDate { get; set; }
        public DateTime? FinishDate { get; set; }
        public bool IsFinish { get; set; }
    }
}

File: SmartTasks.Frontend/src/app/models/smartTask.ts
export class SmartTask{
  //constructor();

  constructor(id: number, name: string, descryption: string, createDate: Date, finishDate: Date, isFinish: boolean, status: number){
    this.id = id;
    this.name = name;
    this.descryption = descryption;
    this.createDate = createDate;
    this.finishDate = finishDate;
    this.isFinish = isFinish;
    this.status = status;
  }

  id: number;
  name: string;
  descryption: string;
  createDate: Date;
  finishDate: Date;
  isFinish: boolean;
  status: number;
}

export interface ISmartTask{
  id: number;
  name: string;
  descryption: string;
  createDate: Date;
  finishDate: Date;
  isFinish: boolean;
  status: number;
}
